### PR TITLE
Fix trailing tabs in comments.

### PR DESCRIPTION
**Pull request by stefanwalther**
Hi,

please accept the following pull request:

- First added a minimal .editorconfig to not destroy your formatting
- Fixed trailing tabs in comments if using tabs as indent-style in your editor
- Updated to a newer babel version, otherwise building on Windows 8.1 failed with "Don't hotlink internal Babel files" (see http://stackoverflow.com/questions/31062714/react-native-error-dont-hotlink-internal-babel-files)

Thx.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[*.py]
```
Comment (h13i32maru) [inline .editorconfig]: ``[*.js]`` typo?

File: package.json [hunk 2]
```diff
@@ -31,12 +31,12 @@
     "marked": "~0.3.3"
   },
   "devDependencies": {
-    "babel": "~5.2.9",
+    "babel": "^5.6.10",
```
Comment (h13i32maru) [inline package.json]: ESDoc head uses babel 5.6.14
Please rebase PR.

File: test/src/UnitTest/CommentParserTest.js [hunk 3]
```diff
@@ -25,6 +25,28 @@ describe('CommentParser:', ()=>{
     assert.deepEqual(tags[4], {tagName: '@tag4', tagValue: 'tag4 value'});
   });
 
+  it('can parse doc comments with trailing tabs', ()=>{
```
Comment (h13i32maru) [inline test/src/UnitTest/CommentParserTest.js]: Please add ``/** @test {CommentParser.parse} */``
https://github.com/h13i32maru/esdoc/pull/17/files#diff-c89ea2e3abbdb4cb289be2624a653f65R6

File: test/src/UnitTest/CommentParserTest.js [hunk 4]
```diff
@@ -25,6 +25,28 @@ describe('CommentParser:', ()=>{
...
+\t* @tag3 tag3 value
+\t* tag3 second value
+\t*
+\t* @tag4 tag4 value
+\t*
+`;
+    let comment = {type: 'Block', value: value};
+    let tags = CommentParser.parse(comment);
+    assert.equal(tags.length, 5);
+    console.log('tags', tags);
```
Comment (h13i32maru) [inline test/src/UnitTest/CommentParserTest.js]: unnecessary log?

**Comment by h13i32maru** [comment]
@stefanwalther Thank you for this PR :smile: 
Please check my comments.

**Comment by stefanwalther** [comment]
All your comments included. thx.

**Pull request closed without merging**